modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/rebalance/DistributionZoneRebalanceEngine.java

```diff
@@ -122,6 +127,12 @@ public DistributionZoneRebalanceEngine(
         this.dataNodesListener = createDistributionZonesDataNodesListener();
         this.partitionsCounterListener = createPartitionsCounterListener();
         this.rebalanceScheduler = rebalanceScheduler;
+        this.distributionZoneRebalanceEngineV2 = new DistributionZoneRebalanceEngineV2(
```

Comment:
If I understood you correctly, we don't have `rebalanceScheduler` in `DistributionZoneRebalanceEngineV2` because it was needed in `DistributionZoneRebalanceEngine` only for counters logic, and we've removed that logic in V2